mass_arm: [200,0,0,0,0,0,
           0,200,0,0,0,0,
           0,0,200,0,0,0,
           0,0,0,200,0,0,
           0,0,0,0,200,0,
           0,0,0,0,0,200]
damping_arm: [300,0,0,0,0,0,
              0,300,0,0,0,0,
              0,0,300,0,0,0,
              0,0,0,300,0,0,
              0,0,0,0,300,0,
              0,0,0,0,0,300]
stiffness_coupling: [100,0,0,0,0,0,
                     0,100,0,0,0,0,
                     0,0,100,0,0,0,
                     0,0,0,100,0,0,
                     0,0,0,0,100,0,
                     0,0,0,0,0,100] # Kp
wrench_stiffness_coupling: [5,0,0,0,0,0,
                     0,5,0,0,0,0,
                     0,0,5,0,0,0,
                     0,0,0,5,0,0,
                     0,0,0,0,5,0,
                     0,0,0,0,0,5] # Kf
damping_car: [15,0,0,0,0,0,
              0,15,0,0,0,0,
              0,0,15,0,0,0,
              0,0,0,15,0,0,
              0,0,0,0,15,0,
              0,0,0,0,0,15]

# base_link:  $(arg tf_prefix)base_link
# end_link:   $(arg tf_prefix)wrist_3_link
# [position (x, y, z)  orientation (x, y, z, w)]
#desired_pose:  [0.4, 0.2, 0.75, 0.707, 0.0, 0.0, 0.707]
#desired_pose:  [0.4, 0.2, 0.75, 1, 0.0, 0.0, 0]

#desired_pose:  [0.4, 0, 0.55, 0.0, 0.707, 0.707, 0.0]
desired_pose:  [0.4, 0.34, 0.65, 0.0, 0.707, 0.707, 0.0]
# 0.55 is the touch area

# reference_force: [0, 0, 5, 0, 0, 0] (N)
# delta_x = -Kf/Kp * force = [0 0 -0.5 0 0 0]
workspace_limits: [-0.50, 0.50, 0.25, 0.80, 0.30, 0.75]

arm_max_vel: 1.5
arm_max_acc: 1.0
platform_max_vel: 0.1
platform_max_acc: 0.2
